azerothcore/azerothcore-wotlk
# fix(DB/AQ20): Ossirian Loot

Corrected Ossirian boss loot

## Changes Proposed:
I made Ossirian drop:
3 books
1 hilt or ring. More likely to drop epic
2 epic non-set
1 head
1 enchanting recipe with 7% chance

I put them in seperate ref tables so there's no duplicate non-set or book drops.

## Issues Addressed:

- Closes #12699
- https://github.com/chromiecraft/chromiecraft/issues/3914

## SOURCE:


## Tests Performed:


- 
![1269_ossirian_loot](https://user-images.githubusercontent.com/74299960/184182621-729f2146-53cb-400d-be6c-c60fd1a6ab13.png)


## How to Test the Changes:

Go to Ossirian
```
.go c 144601
```
Reset Instance
```
.instance u all 
```
1. TP to Ossirrian
2. Kill
3. Check loot
4. Reset 

Also check the loot table in SQL with Keira3. Ossirian (id: 15339) to see if I missed a skill book or item.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
